lib/views/configuration-view.js

```diff
@@ -893,6 +906,7 @@ export default class ConfigurationView extends View {
     }
 
     folderConfigurationView.on('close', (e) => {
+      self.createControlsFolderSelect();
```

Comment:
Unfortunately, reloading the options causes the previous selection from the current selected server will be lost.